lib/components/form/mode-buttons.js

```diff
@@ -54,19 +55,23 @@ const CheckMarkIcon = styled(Icon)`
 
 const ModeButton = ({className, item, onClick, selected}) => {
   const {ModeIcon} = useContext(ComponentContext)
+  const {icon, label, mode} = item
   return (
-    <button
-      className={className}
-      onClick={() => onClick(item.mode)}
-      // FIXME: add hover dropdown?
-      title={item.label}
+    <OverlayTrigger
+      overlay={<Tooltip id={mode}>{label}</Tooltip>}
+      placement='top'
```

Comment:
I think it's a bit more natural to have these appear below the button.

```suggestion
      placement='bottom'
```

Reply:
Sure